client/src/app/site/pages/organization/pages/committees/pages/committee-detail/modules/committee-detail-view/components/committee-detail-view/committee-detail-view.component.ts

```diff
@@ -30,12 +30,8 @@ export class CommitteeDetailViewComponent extends BaseUiComponent {
     public forwardingExpanded = false;
     public requireDuplicateFrom = false;
 
-    public get canManageMeetingsInCommittee(): boolean {
-        return this.operator.hasCommitteePermissionsNonAdminCheck(this.committeeId, CML.can_manage);
-    }
-
-    public get canManageCommittee(): boolean {
-        return this.operator.hasCommitteePermissions(this.committeeId, CML.can_manage);
+    public get canManageCommitteeOrMeetingsInCommittee(): boolean {
```

Comment:
This is confusing. You are delegating this to a lower level so keep the name simple. 
```suggestion
    public get canManageCommittee(): boolean {
```